src/pages/PingDetail.svelte

```diff
@@ -65,6 +62,13 @@
         <EmailAddresses emails={ping.notification_emails} />
       </td>
     </tr>
+    <tr>
+      <td>BigQuery table</td>
```

Comment:
Let's match the capitalization of the other fields.

```suggestion
      <td>BigQuery Table</td>
```